src/test/java/seedu/address/testutil/TypicalQuestions.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.AddressBook;
+import seedu.address.model.question.Question;
+
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DESCRIPTION_Q1;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DESCRIPTION_Q2;
+
+
+/**
+ * A utility class containing a list of {@code Question} objects to be used in tests.
+ */
+public class TypicalQuestions {
+
+
+//    public static final Index INDEX_FIRST_QUESTION = new Index(1);
```

Comment:
Delete dead code!

Reply:
Oh it's not meant to be dead. I'm still trying to work out how to test markq and unmarkq.